browser/Containers/AppContainer.js

```diff
@@ -1,13 +1,54 @@
 import React, {Component} from 'react';
 import {connect} from 'react-redux'
 import { Link } from 'react-router';
+import firebase from 'firebase';
+
 import HTMLEditor from './HTMLEditor';
-import {updateHTML, updateCSS, updateJS, updateServer, updateDatabase} from '../reducers/code';
+import { updateHTML, updateCSS, updateJS, updateServer, updateDatabase, updateHTMLCSSJS } from '../reducers/code';
+import LoadingButton from './LoadingButton';
 
 class AppContainer extends Component {
   constructor(props){
     super(props)
   }
+
+  componentDidMount() {
```

Comment:
Do we still need componentDidMount? Could we take it out, or put a comment related to why we might want to keep this for future use?